ONSdigital/eq-survey-runner
# EQ-227: Disable Automatic Welsh Switching

**What**
Disable Welsh language switching

**How to test**
Change language in web browser settings and make sure it doesn't change on the screen

**Who can test**
Anyone but @LJBabbage
